client/libs/theme/main.scss

```diff
@@ -2,5 +2,15 @@
 @import '~bootstrap-4-grid';
 @import 'libs/theme/fonts.scss';
 
-html, body { height: 100%; }
-body { margin: 0; font-family: 'Montserrat', "Helvetica Neue", sans-serif; }
+html, body { 
+  height: 100%; 
+}
+
+body { 
+  margin: 0; 
+  font-family: 'Montserrat', "Helvetica Neue", sans-serif; 
+}
+
+.cursor-pointer {
```

Comment:
Let's move this to a `utilities.scss` file under the "theme" folder.
You could then probably just import that file into this one.